Documentation/Contributing/CONTRIBUTING.md

```diff
@@ -6,102 +6,31 @@ For larger submissions, please refer to the [feature contribution guidelines / p
 
 All changes be they small or large, need to adhere to the [MRTK coding standards](CodingGuidelines.md), so please ensure you are familiar with these while developing to avoid delays when the change is being reviewed.
 
-If you have any questions, please reach out on the [HoloLens forums](https://forums.hololens.com/) or the [HoloDevelopers slack](https://holodevelopers.slack.com/). You can easily be granted access to the Slack community via the [automatic invitation sender](https://holodevelopersslack.azurewebsites.net/).
```

Comment:
>automatic [](start = 230, length = 9)

Would it be possible to separate these change out from the rest of these release notes? It feels like the contribution stuff is orthogonal to what we're trying to add here (release notes)

Reply:
per discussion, leaving these changes here to reduce the number of documentation PRs reqiiored for release